### DISCUSSION TITLE
Admins, docs, help

### DISCUSSION THREAD

**Discussion by BahamutRU** [innightwolfsleep/llm_telegram_bot#54] [Q&A] [answered]
Forgive my stupidity, but I can't find the documentation for the project. Therefore, I have the following questions:
1. In what format should the identifier be entered in the file telegram_admins.txt ? Numbers, tag?
2. Which mode should be written in bot_mode= so that the admin can edit it (change the model, character), and the rest could not?

**Comment by innightwolfsleep** [comment, marked as answer]
I should make better documentation for this) 
Also, complicate rules matrix isnt implemented yet.

1. Numbers. You can find you own in telegram_bot\history (you own history file)

2. There is no apropriate mode. Each user have separated history and switch to own character. Preset and model settings is common.
There is two modes similiar what you want.
chat-restricted - restrict any changes for all. Only default character/preset/model settings allowed,
chat - restrict changes of model and preset (common settings) for all excep users in admin list

admin mode - anyone can do anything, include changing of common settings.

**Reply by BahamutRU** [reply]
Thx, chat — my choice!

**Reply by BahamutRU** [reply]
In _chat_ all users can change characters, I add new bot_mode for myself. But maybe you are making a mode where and how you do not need to change users?

**Comment by innightwolfsleep** [comment]
Added new rules matrix. Dict "user_rules".

**Reply by BahamutRU** [reply]
Purrfect! Thx!
